content/en/docs/Configuration/p8s-jaeger-grafana/prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Prometheus"
+description: >
+  This page describes how to configure Prometheus for Kiali.
+---
+
+
+## Prometheus configuration
+
+Kiali *requires* Prometheus to generate the
+[topology graph]({{< relref "../../Features/topology" >}}),
+[show metrics]({{< relref "../../Features/details#metrics" >}}),
+[calculate health]({{< relref "../../Features/health" >}}) and
+for several other features. If Prometheus is missing or Kiali
+can't reach it, Kiali won't work properly.
+
+By default, Kiali assumes that Prometheus is available at the URL of the form
+`http://prometheus.<istio_namespace_name>:9090`, which is the usual case if you
+are using [the Prometheus Istio
+add-on](https://istio.io/latest/docs/ops/integrations/prometheus/#option-1-quick-start).
+If your Prometheus instance has a different service name or is installed to a
+different namespace, you must manually provide the endpoint where it is
+available, like in the following example:
+
+```yaml
+spec:
+  external_services:
+    prometheus:
+      # Prometheus service name is "metrics" and is in the "telemetry" namespace
+      url: "http://metrics.telemetry:9090/"
+```
+
+{{% alert color="success" %}}
+Notice that you don't need to expose Prometheus outside the cluster. It is
+enough to provide the Kubernetes internal service URL.
+{{% /alert %}}
+
+Kiali maintains an internal cache of some Prometheus queries to improve
+performance (mainly, the queries to calculate Health indicators). It
+would be very rare to see data delays, but should you notice any delays you may
+tune caching parameters to values that work better for your environment. These
+are the default values:
```

Comment:
I would prefer not to have default values spread out. Now if we ever change those values, we will have multiple places to remember where to update. We already have a single location defining our defaults - point the reader there. For example:

```
The default values for the Prometheus cache settings can be found in the Kiali CR Reference page
```

Reply:
Again, just existing content, but given the new CR ref page, yes, I agree.  I'll make the change.